src/uikit.cs

```diff
@@ -13652,35 +13654,40 @@ public partial interface UIPopoverPresentationController {
 	[Protocol, Model]
 	[BaseType (typeof (NSObject))]
 	public partial interface UIAdaptivePresentationControllerDelegate {
+		[IgnoredInDelegate]
 		[Export ("adaptivePresentationStyleForPresentationController:")]
 		UIModalPresentationStyle GetAdaptivePresentationStyle (UIPresentationController forPresentationController);
 	
-		[Export ("presentationController:viewControllerForAdaptivePresentationStyle:")]
+		[Export ("presentationController:viewControllerForAdaptivePresentationStyle:"), 
+			DelegateName ("AdaptivePresentationRequestedWithStyle"), DefaultValue (null)]
```

Comment:
Minor but since this will actually generate a delegate type we tend to prefix them just like any other class in the contained namespace i.e. `AdaptivePresentationRequestedWithStyle` would become `UIAdaptivePresentationRequestedWithStyle` 

Also I think you can replace/eliminate the `WithStyle` suffix